tests/tests_integration/test_api/test_datapoints.py

```diff
@@ -184,7 +184,7 @@ def all_retrieve_endpoints(cognite_client, retrieve_endpoints):
 
 
 @pytest.fixture(scope="session")
-def instance_ts_id(cognite_client_alpha: CogniteClient, alpha_test_space: Space) -> InstanceId:
+def instance_ts_id(cognite_client: CogniteClient, alpha_test_space: Space) -> InstanceId:
```

Comment:
is the alpha client no longer used? Should we remove the fixture?

Reply:
Still in use by files tests. Maybe these can also be changed. I decided not to remove it as we will certainly have alpha features in the futures